### PR TITLE
Updates to doc for multicluster todo-list and hello-helidon examples

### PR DESCRIPTION
**Pull request by bobdonat**
This pull request updates the multicluster todo-list and hello-helidon examples.  The examples were changed to not use the MultiClusterComponents and MultiClusterSecrets objects.

### CODE REVIEW

File: content/en/docs/samples/multicluster/hello-helidon/_index.md [hunk 1]
```diff
@@ -49,9 +49,8 @@ Follow the instructions for [troubleshooting]({{< relref "/docs/samples/hello-he
    $ kubectl --kubeconfig $KUBECONFIG_MANAGED1 get namespace hello-helidon
    ```
 
-1. Verify that the multicluster resources for the application all exist.
+1. Verify that the multicluster resource for the application exist.
```
Comment (mgianatagh) [inline content/en/docs/samples/multicluster/hello-helidon/_index.md:52]: "exists"?

**Review by rosemarymarano — APPROVED** [review]

File: content/en/docs/samples/multicluster/hello-helidon/_index.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Apply the VerrazzanoProject resource on the admin cluster that defines the names
 1. Apply the `hello-helidon` multicluster resources to deploy the application.  Each multicluster resource is an envelope that contains an OAM resource and a list of clusters to which to deploy.
```
Comment (desagar) [inline content/en/docs/samples/multicluster/hello-helidon/_index.md:23]: this line may need editing too. It seems to imply that everything we are applying is a "multicluster resource"

File: content/en/docs/samples/multicluster/hello-helidon/_index.md [hunk 3]
```diff
@@ -66,22 +65,15 @@ By default, the application is located on the managed cluster called `managed1`.
    ```
    This environment variable is used in subsequent steps.
 
-1. To change their placement, patch the `hello-helidon` multicluster resources.
+1. To change their placement, patch the `hello-helidon` multicluster resource.
```
Comment (desagar) [inline content/en/docs/samples/multicluster/hello-helidon/_index.md:68]: we are only patching the the app config - so maybe this should say patch the hello-helidon multicluster application configuration.

File: content/en/docs/samples/multicluster/hello-helidon/_index.md [hunk 4]
```diff
@@ -66,22 +65,15 @@ By default, the application is located on the managed cluster called `managed1`.
...
        -n hello-helidon \
        --type merge \
        --patch "$(cat $CHANGE_PLACEMENT_PATCH_FILE)"
-   $ kubectl --kubeconfig $KUBECONFIG_ADMIN patch mccomp hello-helidon-component \
-       -n hello-helidon \
-       --type merge \
-       --patch "$(cat $CHANGE_PLACEMENT_PATCH_FILE)"
    ```
-1. To verify that their placement has changed, view the multicluster resources.
+1. To verify that their placement has changed, view the multicluster resource.
```
Comment (desagar) [inline content/en/docs/samples/multicluster/hello-helidon/_index.md:75]: their => its

File: content/en/docs/samples/multicluster/todo-list/_index.md [hunk 5]
```diff
@@ -194,3 +193,20 @@ multicluster secret resource will generate the required secret in the `mc-todo-l
    mysql-5c75c8b7f-vlhck    1/1     Running   0          19h
    tododomain-adminserver   2/2     Running   0          19h
    ```
+
+## Undeploy the ToDo List application
+
+Regardless of its location, to undeploy the application,
```
Comment (desagar) [inline content/en/docs/samples/multicluster/todo-list/_index.md:199]: should we say here that this undeploy is for all clusters in which the application is located?

Comment (bobdonat) [reply content/en/docs/samples/multicluster/todo-list/_index.md:199]: I can do that.  This was a cut and paste from the hello-helidon multi-cluster example

File: content/en/docs/samples/multicluster/hello-helidon/_index.md (hunk 4 above)
Comment (bobdonat) [reply content/en/docs/samples/multicluster/hello-helidon/_index.md:75]: will fix

File: content/en/docs/samples/multicluster/hello-helidon/_index.md (hunk 3 above)
Comment (bobdonat) [reply content/en/docs/samples/multicluster/hello-helidon/_index.md:68]: will fix

File: content/en/docs/samples/multicluster/hello-helidon/_index.md (hunk 2 above)
Comment (bobdonat) [reply content/en/docs/samples/multicluster/hello-helidon/_index.md:23]: will fix

File: content/en/docs/samples/multicluster/hello-helidon/_index.md (hunk 1 above)
Comment (bobdonat) [reply content/en/docs/samples/multicluster/hello-helidon/_index.md:52]: will fix

File: content/en/docs/samples/multicluster/hello-helidon/_index.md (hunk 4 above)
Comment (bobdonat) [reply content/en/docs/samples/multicluster/hello-helidon/_index.md]: fixed

File: content/en/docs/samples/multicluster/hello-helidon/_index.md (hunk 3 above)
Comment (bobdonat) [reply content/en/docs/samples/multicluster/hello-helidon/_index.md]: fixed

File: content/en/docs/samples/multicluster/hello-helidon/_index.md (hunk 2 above)
Comment (bobdonat) [reply content/en/docs/samples/multicluster/hello-helidon/_index.md]: fixed

File: content/en/docs/samples/multicluster/hello-helidon/_index.md (hunk 1 above)
Comment (bobdonat) [reply content/en/docs/samples/multicluster/hello-helidon/_index.md]: fixed

**Review by desagar — APPROVED** [review]

**Pull request merged**